src/css/tickets.css

```diff
@@ -0,0 +1,120 @@
+/*Tickets page CSS*/
+
+
+ /* announcements bar */
+.announcements-container {
+  display: flex;
+  align-items: center;
+  background-color: rgb(231, 211, 31);
+  padding-top: 5px;
+  padding-bottom: 5px;
+}
+
+.announcements-container > div {
```

Comment:
Maybe try to give the `announcements-container`  children a class of itself like `.announcements-container-child`